[2024-11-11T00:18:29.397+0000] {processor.py:161} INFO - Started process (PID=59) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:18:29.398+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:18:29.405+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:18:29.405+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:18:34.546+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:18:34.535+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 11, in <module>
    import mlflow.keras
  File "/home/airflow/.local/lib/python3.12/site-packages/mlflow/keras/__init__.py", line 2, in <module>
    import keras
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/__init__.py", line 4, in <module>
    from keras.api import DTypePolicy
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/api/__init__.py", line 8, in <module>
    from keras.api import activations
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/api/activations/__init__.py", line 7, in <module>
    from keras.src.activations import deserialize
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/__init__.py", line 1, in <module>
    from keras.src import activations
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/activations/__init__.py", line 3, in <module>
    from keras.src.activations.activations import elu
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/activations/activations.py", line 1, in <module>
    from keras.src import backend
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/__init__.py", line 9, in <module>
    from keras.src.backend.common.dtypes import result_type
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/common/__init__.py", line 2, in <module>
    from keras.src.backend.common.dtypes import result_type
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/common/dtypes.py", line 5, in <module>
    from keras.src.backend.common.variables import standardize_dtype
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/common/variables.py", line 11, in <module>
    from keras.src.utils.module_utils import tensorflow as tf
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/utils/__init__.py", line 1, in <module>
    from keras.src.utils.audio_dataset_utils import audio_dataset_from_directory
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/utils/audio_dataset_utils.py", line 4, in <module>
    from keras.src.utils import dataset_utils
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/utils/dataset_utils.py", line 9, in <module>
    from keras.src import tree
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/tree/__init__.py", line 1, in <module>
    from keras.src.tree.tree_api import assert_same_structure
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/tree/tree_api.py", line 6, in <module>
    from keras.src.tree import optree_impl as tree_impl
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/tree/optree_impl.py", line 17, in <module>
    from tensorflow.python.trackable.data_structures import ListWrapper
ModuleNotFoundError: No module named 'tensorflow.python'
[2024-11-11T00:18:34.548+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:18:34.565+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 5.176 seconds
[2024-11-11T00:19:04.764+0000] {processor.py:161} INFO - Started process (PID=88) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:19:04.768+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:19:04.771+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:19:04.771+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:19:06.056+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:19:06.053+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 11, in <module>
    import mlflow.keras
  File "/home/airflow/.local/lib/python3.12/site-packages/mlflow/keras/__init__.py", line 2, in <module>
    import keras
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/__init__.py", line 4, in <module>
    from keras.api import DTypePolicy
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/api/__init__.py", line 8, in <module>
    from keras.api import activations
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/api/activations/__init__.py", line 7, in <module>
    from keras.src.activations import deserialize
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/__init__.py", line 1, in <module>
    from keras.src import activations
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/activations/__init__.py", line 3, in <module>
    from keras.src.activations.activations import elu
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/activations/activations.py", line 1, in <module>
    from keras.src import backend
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/__init__.py", line 9, in <module>
    from keras.src.backend.common.dtypes import result_type
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/common/__init__.py", line 2, in <module>
    from keras.src.backend.common.dtypes import result_type
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/common/dtypes.py", line 5, in <module>
    from keras.src.backend.common.variables import standardize_dtype
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/common/variables.py", line 11, in <module>
    from keras.src.utils.module_utils import tensorflow as tf
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/utils/__init__.py", line 1, in <module>
    from keras.src.utils.audio_dataset_utils import audio_dataset_from_directory
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/utils/audio_dataset_utils.py", line 4, in <module>
    from keras.src.utils import dataset_utils
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/utils/dataset_utils.py", line 9, in <module>
    from keras.src import tree
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/tree/__init__.py", line 1, in <module>
    from keras.src.tree.tree_api import assert_same_structure
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/tree/tree_api.py", line 6, in <module>
    from keras.src.tree import optree_impl as tree_impl
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/tree/optree_impl.py", line 17, in <module>
    from tensorflow.python.trackable.data_structures import ListWrapper
ModuleNotFoundError: No module named 'tensorflow.python'
[2024-11-11T00:19:06.056+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:19:06.064+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.313 seconds
[2024-11-11T00:19:36.339+0000] {processor.py:161} INFO - Started process (PID=112) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:19:36.341+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:19:36.343+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:19:36.343+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:19:37.855+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:19:37.852+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 11, in <module>
    import mlflow.keras
  File "/home/airflow/.local/lib/python3.12/site-packages/mlflow/keras/__init__.py", line 2, in <module>
    import keras
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/__init__.py", line 4, in <module>
    from keras.api import DTypePolicy
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/api/__init__.py", line 8, in <module>
    from keras.api import activations
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/api/activations/__init__.py", line 7, in <module>
    from keras.src.activations import deserialize
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/__init__.py", line 1, in <module>
    from keras.src import activations
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/activations/__init__.py", line 3, in <module>
    from keras.src.activations.activations import elu
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/activations/activations.py", line 1, in <module>
    from keras.src import backend
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/__init__.py", line 9, in <module>
    from keras.src.backend.common.dtypes import result_type
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/common/__init__.py", line 2, in <module>
    from keras.src.backend.common.dtypes import result_type
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/common/dtypes.py", line 5, in <module>
    from keras.src.backend.common.variables import standardize_dtype
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/common/variables.py", line 11, in <module>
    from keras.src.utils.module_utils import tensorflow as tf
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/utils/__init__.py", line 1, in <module>
    from keras.src.utils.audio_dataset_utils import audio_dataset_from_directory
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/utils/audio_dataset_utils.py", line 4, in <module>
    from keras.src.utils import dataset_utils
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/utils/dataset_utils.py", line 9, in <module>
    from keras.src import tree
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/tree/__init__.py", line 1, in <module>
    from keras.src.tree.tree_api import assert_same_structure
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/tree/tree_api.py", line 6, in <module>
    from keras.src.tree import optree_impl as tree_impl
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/tree/optree_impl.py", line 17, in <module>
    from tensorflow.python.trackable.data_structures import ListWrapper
ModuleNotFoundError: No module named 'tensorflow.python'
[2024-11-11T00:19:37.856+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:19:37.866+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.534 seconds
[2024-11-11T00:20:08.059+0000] {processor.py:161} INFO - Started process (PID=136) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:20:08.061+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:20:08.065+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:20:08.065+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:20:09.214+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:20:09.211+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 11, in <module>
    import mlflow.keras
  File "/home/airflow/.local/lib/python3.12/site-packages/mlflow/keras/__init__.py", line 2, in <module>
    import keras
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/__init__.py", line 4, in <module>
    from keras.api import DTypePolicy
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/api/__init__.py", line 8, in <module>
    from keras.api import activations
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/api/activations/__init__.py", line 7, in <module>
    from keras.src.activations import deserialize
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/__init__.py", line 1, in <module>
    from keras.src import activations
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/activations/__init__.py", line 3, in <module>
    from keras.src.activations.activations import elu
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/activations/activations.py", line 1, in <module>
    from keras.src import backend
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/__init__.py", line 9, in <module>
    from keras.src.backend.common.dtypes import result_type
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/common/__init__.py", line 2, in <module>
    from keras.src.backend.common.dtypes import result_type
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/common/dtypes.py", line 5, in <module>
    from keras.src.backend.common.variables import standardize_dtype
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/common/variables.py", line 11, in <module>
    from keras.src.utils.module_utils import tensorflow as tf
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/utils/__init__.py", line 1, in <module>
    from keras.src.utils.audio_dataset_utils import audio_dataset_from_directory
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/utils/audio_dataset_utils.py", line 4, in <module>
    from keras.src.utils import dataset_utils
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/utils/dataset_utils.py", line 9, in <module>
    from keras.src import tree
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/tree/__init__.py", line 1, in <module>
    from keras.src.tree.tree_api import assert_same_structure
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/tree/tree_api.py", line 6, in <module>
    from keras.src.tree import optree_impl as tree_impl
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/tree/optree_impl.py", line 17, in <module>
    from tensorflow.python.trackable.data_structures import ListWrapper
ModuleNotFoundError: No module named 'tensorflow.python'
[2024-11-11T00:20:09.215+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:20:09.223+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.190 seconds
[2024-11-11T00:20:39.537+0000] {processor.py:161} INFO - Started process (PID=160) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:20:39.539+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:20:39.542+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:20:39.542+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:20:40.617+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:20:40.615+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 11, in <module>
    import mlflow.keras
  File "/home/airflow/.local/lib/python3.12/site-packages/mlflow/keras/__init__.py", line 2, in <module>
    import keras
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/__init__.py", line 4, in <module>
    from keras.api import DTypePolicy
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/api/__init__.py", line 8, in <module>
    from keras.api import activations
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/api/activations/__init__.py", line 7, in <module>
    from keras.src.activations import deserialize
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/__init__.py", line 1, in <module>
    from keras.src import activations
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/activations/__init__.py", line 3, in <module>
    from keras.src.activations.activations import elu
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/activations/activations.py", line 1, in <module>
    from keras.src import backend
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/__init__.py", line 9, in <module>
    from keras.src.backend.common.dtypes import result_type
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/common/__init__.py", line 2, in <module>
    from keras.src.backend.common.dtypes import result_type
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/common/dtypes.py", line 5, in <module>
    from keras.src.backend.common.variables import standardize_dtype
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/backend/common/variables.py", line 11, in <module>
    from keras.src.utils.module_utils import tensorflow as tf
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/utils/__init__.py", line 1, in <module>
    from keras.src.utils.audio_dataset_utils import audio_dataset_from_directory
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/utils/audio_dataset_utils.py", line 4, in <module>
    from keras.src.utils import dataset_utils
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/utils/dataset_utils.py", line 9, in <module>
    from keras.src import tree
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/tree/__init__.py", line 1, in <module>
    from keras.src.tree.tree_api import assert_same_structure
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/tree/tree_api.py", line 6, in <module>
    from keras.src.tree import optree_impl as tree_impl
  File "/home/airflow/.local/lib/python3.12/site-packages/keras/src/tree/optree_impl.py", line 17, in <module>
    from tensorflow.python.trackable.data_structures import ListWrapper
ModuleNotFoundError: No module named 'tensorflow.python'
[2024-11-11T00:20:40.618+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:20:40.626+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.103 seconds
[2024-11-11T00:21:49.461+0000] {processor.py:161} INFO - Started process (PID=101) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:21:49.466+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:21:49.472+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:21:49.471+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:21:51.349+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:21:51.346+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 3, in <module>
    import tensorflow as tf
ModuleNotFoundError: No module named 'tensorflow'
[2024-11-11T00:21:51.350+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:21:51.385+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.937 seconds
[2024-11-11T00:22:21.850+0000] {processor.py:161} INFO - Started process (PID=119) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:22:21.852+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:22:21.859+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:22:21.859+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:22:22.370+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:22:22.369+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 3, in <module>
    import tensorflow as tf
ModuleNotFoundError: No module named 'tensorflow'
[2024-11-11T00:22:22.370+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:22:22.395+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.564 seconds
[2024-11-11T00:22:52.703+0000] {processor.py:161} INFO - Started process (PID=136) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:22:52.704+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:22:52.707+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:22:52.707+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:22:53.135+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:22:53.134+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 3, in <module>
    import tensorflow as tf
ModuleNotFoundError: No module named 'tensorflow'
[2024-11-11T00:22:53.135+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:22:53.150+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.456 seconds
[2024-11-11T00:23:23.544+0000] {processor.py:161} INFO - Started process (PID=153) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:23:23.545+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:23:23.548+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:23:23.547+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:23:23.926+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:23:23.924+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 3, in <module>
    import tensorflow as tf
ModuleNotFoundError: No module named 'tensorflow'
[2024-11-11T00:23:23.926+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:23:23.935+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.400 seconds
[2024-11-11T00:23:54.972+0000] {processor.py:161} INFO - Started process (PID=170) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:23:54.974+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:23:54.976+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:23:54.976+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:23:55.357+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:23:55.356+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 3, in <module>
    import tensorflow as tf
ModuleNotFoundError: No module named 'tensorflow'
[2024-11-11T00:23:55.358+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:23:55.370+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.406 seconds
[2024-11-11T00:23:56.399+0000] {processor.py:161} INFO - Started process (PID=172) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:23:56.399+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:23:56.401+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:23:56.401+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:23:56.609+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:23:56.608+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 3, in <module>
    import tensorflow as tf
ModuleNotFoundError: No module named 'tensorflow'
[2024-11-11T00:23:56.609+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:23:56.616+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.223 seconds
[2024-11-11T00:28:37.443+0000] {processor.py:161} INFO - Started process (PID=76) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:28:37.454+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:28:37.473+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:28:37.472+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:28:41.938+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:28:41.893+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 3, in <module>
    import tensorflow as tf
  File "/home/airflow/.local/lib/python3.12/site-packages/tensorflow/__init__.py", line 40, in <module>
    from tensorflow.python import pywrap_tensorflow as _pywrap_tensorflow  # pylint: disable=unused-import
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tensorflow/python/pywrap_tensorflow.py", line 29, in <module>
    from tensorflow.python.platform import self_check
ModuleNotFoundError: No module named 'tensorflow.python.platform'
[2024-11-11T00:28:41.941+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:28:42.057+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 4.718 seconds
[2024-11-11T00:29:12.712+0000] {processor.py:161} INFO - Started process (PID=95) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:29:12.758+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:29:12.850+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:29:12.838+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:29:20.141+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:29:20.125+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 3, in <module>
    import tensorflow as tf
  File "/home/airflow/.local/lib/python3.12/site-packages/tensorflow/__init__.py", line 40, in <module>
    from tensorflow.python import pywrap_tensorflow as _pywrap_tensorflow  # pylint: disable=unused-import
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tensorflow/python/pywrap_tensorflow.py", line 29, in <module>
    from tensorflow.python.platform import self_check
ModuleNotFoundError: No module named 'tensorflow.python.platform'
[2024-11-11T00:29:20.142+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:29:20.181+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 7.798 seconds
[2024-11-11T00:29:50.631+0000] {processor.py:161} INFO - Started process (PID=112) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:29:50.633+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:29:50.641+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:29:50.638+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:29:51.630+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:29:51.627+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 3, in <module>
    import tensorflow as tf
  File "/home/airflow/.local/lib/python3.12/site-packages/tensorflow/__init__.py", line 40, in <module>
    from tensorflow.python import pywrap_tensorflow as _pywrap_tensorflow  # pylint: disable=unused-import
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tensorflow/python/pywrap_tensorflow.py", line 29, in <module>
    from tensorflow.python.platform import self_check
ModuleNotFoundError: No module named 'tensorflow.python.platform'
[2024-11-11T00:29:51.630+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:29:51.650+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.029 seconds
[2024-11-11T00:30:22.002+0000] {processor.py:161} INFO - Started process (PID=129) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:30:22.004+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:30:22.006+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:30:22.006+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:30:22.500+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:30:22.499+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 3, in <module>
    import tensorflow as tf
  File "/home/airflow/.local/lib/python3.12/site-packages/tensorflow/__init__.py", line 40, in <module>
    from tensorflow.python import pywrap_tensorflow as _pywrap_tensorflow  # pylint: disable=unused-import
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tensorflow/python/pywrap_tensorflow.py", line 29, in <module>
    from tensorflow.python.platform import self_check
ModuleNotFoundError: No module named 'tensorflow.python.platform'
[2024-11-11T00:30:22.501+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:30:22.514+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.523 seconds
[2024-11-11T00:30:52.577+0000] {processor.py:161} INFO - Started process (PID=146) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:30:52.578+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:30:52.581+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:30:52.581+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:30:53.131+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:30:53.129+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 3, in <module>
    import tensorflow as tf
  File "/home/airflow/.local/lib/python3.12/site-packages/tensorflow/__init__.py", line 40, in <module>
    from tensorflow.python import pywrap_tensorflow as _pywrap_tensorflow  # pylint: disable=unused-import
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tensorflow/python/pywrap_tensorflow.py", line 29, in <module>
    from tensorflow.python.platform import self_check
ModuleNotFoundError: No module named 'tensorflow.python.platform'
[2024-11-11T00:30:53.131+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:30:53.144+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.573 seconds
[2024-11-11T00:31:23.984+0000] {processor.py:161} INFO - Started process (PID=163) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:31:23.986+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:31:23.988+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:31:23.988+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:31:24.674+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:31:24.671+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 3, in <module>
    import tensorflow as tf
  File "/home/airflow/.local/lib/python3.12/site-packages/tensorflow/__init__.py", line 40, in <module>
    from tensorflow.python import pywrap_tensorflow as _pywrap_tensorflow  # pylint: disable=unused-import
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tensorflow/python/pywrap_tensorflow.py", line 29, in <module>
    from tensorflow.python.platform import self_check
ModuleNotFoundError: No module named 'tensorflow.python.platform'
[2024-11-11T00:31:24.675+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:31:24.718+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.744 seconds
[2024-11-11T00:31:54.779+0000] {processor.py:161} INFO - Started process (PID=180) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:31:54.780+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:31:54.785+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:31:54.782+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:31:55.264+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:31:55.263+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 3, in <module>
    import tensorflow as tf
  File "/home/airflow/.local/lib/python3.12/site-packages/tensorflow/__init__.py", line 40, in <module>
    from tensorflow.python import pywrap_tensorflow as _pywrap_tensorflow  # pylint: disable=unused-import
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tensorflow/python/pywrap_tensorflow.py", line 29, in <module>
    from tensorflow.python.platform import self_check
ModuleNotFoundError: No module named 'tensorflow.python.platform'
[2024-11-11T00:31:55.265+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:31:55.275+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.505 seconds
[2024-11-11T00:32:25.402+0000] {processor.py:161} INFO - Started process (PID=197) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:32:25.404+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:32:25.408+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:32:25.408+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:32:26.112+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:32:26.109+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 14, in <module>
    from dags.ModelDevelopment.LSTM import main as LSTM
  File "/opt/airflow/dags/ModelDevelopment/LSTM.py", line 3, in <module>
    import tensorflow as tf
  File "/home/airflow/.local/lib/python3.12/site-packages/tensorflow/__init__.py", line 40, in <module>
    from tensorflow.python import pywrap_tensorflow as _pywrap_tensorflow  # pylint: disable=unused-import
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tensorflow/python/pywrap_tensorflow.py", line 29, in <module>
    from tensorflow.python.platform import self_check
ModuleNotFoundError: No module named 'tensorflow.python.platform'
[2024-11-11T00:32:26.112+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:32:26.143+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.755 seconds
[2024-11-11T00:32:54.651+0000] {processor.py:161} INFO - Started process (PID=214) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:32:54.653+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:32:54.657+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:32:54.657+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:32:56.357+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:32:56.355+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 15, in <module>
    from dags.ModelDevelopment.Prophet import main as prophet
  File "/opt/airflow/dags/ModelDevelopment/Prophet.py", line 6, in <module>
    from prophet import Prophet
ModuleNotFoundError: No module named 'prophet'
[2024-11-11T00:32:56.358+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:32:56.441+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.803 seconds
[2024-11-11T00:33:03.716+0000] {processor.py:161} INFO - Started process (PID=232) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:33:03.717+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:33:03.719+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:33:03.718+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:33:04.596+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:33:04.595+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 15, in <module>
    from dags.ModelDevelopment.Prophet import main as prophet
  File "/opt/airflow/dags/ModelDevelopment/Prophet.py", line 6, in <module>
    from prophet import Prophet
ModuleNotFoundError: No module named 'prophet'
[2024-11-11T00:33:04.597+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:33:04.623+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.913 seconds
[2024-11-11T00:33:21.916+0000] {processor.py:161} INFO - Started process (PID=247) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:33:21.918+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:33:21.922+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:33:21.921+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:33:22.825+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:33:22.824+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 15, in <module>
    from dags.ModelDevelopment.Prophet import main as prophet
  File "/opt/airflow/dags/ModelDevelopment/Prophet.py", line 6, in <module>
    from prophet import Prophet
ModuleNotFoundError: No module named 'prophet'
[2024-11-11T00:33:22.825+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:33:22.845+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.940 seconds
[2024-11-11T00:34:10.848+0000] {processor.py:161} INFO - Started process (PID=65) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:34:10.850+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:34:10.852+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:34:10.852+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:34:13.021+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:34:13.018+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 15, in <module>
    from dags.ModelDevelopment.Prophet import main as prophet
  File "/opt/airflow/dags/ModelDevelopment/Prophet.py", line 6, in <module>
    from prophet import Prophet
ModuleNotFoundError: No module named 'prophet'
[2024-11-11T00:34:13.022+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:34:13.064+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 2.222 seconds
[2024-11-11T00:34:43.423+0000] {processor.py:161} INFO - Started process (PID=89) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:34:43.424+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:34:43.429+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:34:43.428+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:34:45.042+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:34:45.040+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 15, in <module>
    from dags.ModelDevelopment.Prophet import main as prophet
  File "/opt/airflow/dags/ModelDevelopment/Prophet.py", line 6, in <module>
    from prophet import Prophet
ModuleNotFoundError: No module named 'prophet'
[2024-11-11T00:34:45.042+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:34:45.063+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.654 seconds
[2024-11-11T00:35:15.493+0000] {processor.py:161} INFO - Started process (PID=119) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:35:15.495+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:35:15.498+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:35:15.497+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:35:16.532+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:35:16.521+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 15, in <module>
    from dags.ModelDevelopment.Prophet import main as prophet
  File "/opt/airflow/dags/ModelDevelopment/Prophet.py", line 6, in <module>
    from prophet import Prophet
ModuleNotFoundError: No module named 'prophet'
[2024-11-11T00:35:16.532+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:35:16.547+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.064 seconds
[2024-11-11T00:35:46.936+0000] {processor.py:161} INFO - Started process (PID=143) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:35:46.939+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:35:46.943+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:35:46.943+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:35:47.793+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:35:47.786+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 15, in <module>
    from dags.ModelDevelopment.Prophet import main as prophet
  File "/opt/airflow/dags/ModelDevelopment/Prophet.py", line 6, in <module>
    from prophet import Prophet
ModuleNotFoundError: No module named 'prophet'
[2024-11-11T00:35:47.794+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:35:47.810+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.886 seconds
[2024-11-11T00:37:33.237+0000] {processor.py:161} INFO - Started process (PID=110) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:37:33.252+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:37:33.279+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:37:33.276+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:37:37.799+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:37:37.798+0000] {font_manager.py:1584} INFO - generated new fontManager
[2024-11-11T00:37:38.005+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:37:37.999+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 15, in <module>
    from dags.ModelDevelopment.Prophet import main as prophet
  File "/opt/airflow/dags/ModelDevelopment/Prophet.py", line 6, in <module>
    from prophet import Prophet
ModuleNotFoundError: No module named 'prophet'
[2024-11-11T00:37:38.005+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:37:38.068+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 4.871 seconds
[2024-11-11T00:38:08.920+0000] {processor.py:161} INFO - Started process (PID=137) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:38:08.921+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:38:08.924+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:38:08.923+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:38:10.834+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:38:10.827+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 15, in <module>
    from dags.ModelDevelopment.Prophet import main as prophet
  File "/opt/airflow/dags/ModelDevelopment/Prophet.py", line 6, in <module>
    from prophet import Prophet
ModuleNotFoundError: No module named 'prophet'
[2024-11-11T00:38:10.834+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:38:10.861+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.947 seconds
[2024-11-11T00:40:12.322+0000] {processor.py:161} INFO - Started process (PID=113) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:40:12.328+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:40:12.332+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:40:12.332+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:40:16.056+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:40:16.055+0000] {font_manager.py:1584} INFO - generated new fontManager
[2024-11-11T00:40:16.667+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:40:16.667+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:40:17.520+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:40:17.509+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 44, in <module>
    run_Prophet = PythonOperator(
                  ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 220, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 893, in __init__
    validate_key(task_id)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 57, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'Prophet model' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2024-11-11T00:40:17.521+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:40:17.550+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 5.245 seconds
[2024-11-11T00:40:47.813+0000] {processor.py:161} INFO - Started process (PID=142) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:40:47.817+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:40:47.826+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:40:47.824+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:40:50.433+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:40:50.432+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:40:50.787+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:40:50.785+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script_model.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script_model.py", line 44, in <module>
    run_Prophet = PythonOperator(
                  ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 220, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 893, in __init__
    validate_key(task_id)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 57, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'Prophet model' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2024-11-11T00:40:50.787+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:40:50.796+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 3.018 seconds
[2024-11-11T00:43:25.823+0000] {processor.py:161} INFO - Started process (PID=66) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:43:25.829+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:43:25.842+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:43:25.840+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:43:29.345+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:43:29.341+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:43:31.084+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:43:31.297+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:43:31.296+0000] {override.py:1829} INFO - Created Permission View: can edit on DAG:modeling_pipeline
[2024-11-11T00:43:31.303+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:43:31.303+0000] {override.py:1829} INFO - Created Permission View: can delete on DAG:modeling_pipeline
[2024-11-11T00:43:31.306+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:43:31.306+0000] {override.py:1829} INFO - Created Permission View: can read on DAG:modeling_pipeline
[2024-11-11T00:43:31.306+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:43:31.306+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:43:31.317+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:43:31.316+0000] {dag.py:3118} INFO - Creating ORM DAG for modeling_pipeline
[2024-11-11T00:43:31.318+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:43:31.318+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:43:31.337+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 5.548 seconds
[2024-11-11T00:44:01.654+0000] {processor.py:161} INFO - Started process (PID=100) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:44:01.655+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:44:01.656+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:44:01.656+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:44:02.416+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:44:02.416+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:44:02.672+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:44:02.685+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:44:02.685+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:44:02.691+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:44:02.691+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:44:02.698+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.049 seconds
[2024-11-11T00:44:32.950+0000] {processor.py:161} INFO - Started process (PID=116) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:44:32.951+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:44:32.954+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:44:32.954+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:44:34.002+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:44:34.002+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:44:34.282+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:44:34.294+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:44:34.294+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:44:34.302+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:44:34.302+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:44:34.309+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.368 seconds
[2024-11-11T00:45:04.485+0000] {processor.py:161} INFO - Started process (PID=141) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:45:04.487+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:45:04.490+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:45:04.489+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:45:05.388+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:45:05.388+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:45:05.633+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:45:05.644+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:45:05.643+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:45:05.648+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:45:05.648+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:45:05.655+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.180 seconds
[2024-11-11T00:45:35.864+0000] {processor.py:161} INFO - Started process (PID=166) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:45:35.866+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:45:35.868+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:45:35.868+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:45:36.720+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:45:36.720+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:45:36.976+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:45:36.987+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:45:36.986+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:45:36.991+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:45:36.991+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:45:36.997+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.149 seconds
[2024-11-11T00:46:07.289+0000] {processor.py:161} INFO - Started process (PID=191) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:46:07.291+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:46:07.294+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:46:07.294+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:46:08.253+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:46:08.253+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:46:08.556+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:46:08.568+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:46:08.567+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:46:08.574+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:46:08.573+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:46:08.580+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.305 seconds
[2024-11-11T00:46:38.826+0000] {processor.py:161} INFO - Started process (PID=216) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:46:38.828+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:46:38.831+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:46:38.831+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:46:39.676+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:46:39.675+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:46:39.928+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:46:39.940+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:46:39.939+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:46:39.945+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:46:39.945+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:46:39.954+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.144 seconds
[2024-11-11T00:47:10.257+0000] {processor.py:161} INFO - Started process (PID=241) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:47:10.260+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:47:10.264+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:47:10.264+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:47:11.413+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:47:11.413+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:47:11.758+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:47:11.781+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:47:11.780+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:47:11.786+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:47:11.786+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:47:11.794+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.547 seconds
[2024-11-11T00:47:41.898+0000] {processor.py:161} INFO - Started process (PID=266) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:47:41.900+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:47:41.902+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:47:41.902+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:47:42.834+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:47:42.834+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:47:43.072+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:47:43.085+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:47:43.085+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:47:43.090+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:47:43.090+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:47:43.097+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.209 seconds
[2024-11-11T00:48:13.243+0000] {processor.py:161} INFO - Started process (PID=291) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:48:13.244+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:48:13.248+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:48:13.248+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:48:18.334+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:48:18.333+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:48:19.110+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:48:19.138+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:48:19.137+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:48:19.154+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:48:19.154+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:48:19.166+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 5.933 seconds
[2024-11-11T00:48:49.457+0000] {processor.py:161} INFO - Started process (PID=325) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:48:49.462+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:48:49.466+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:48:49.466+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:48:50.358+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:48:50.358+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:48:50.649+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:48:50.665+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:48:50.665+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:48:50.671+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:48:50.671+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:48:50.678+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.229 seconds
[2024-11-11T00:49:20.963+0000] {processor.py:161} INFO - Started process (PID=343) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:49:20.964+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:49:20.965+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:49:20.965+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:49:21.818+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:49:21.818+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:49:22.098+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:49:22.121+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:49:22.121+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:49:22.127+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:49:22.127+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:49:22.135+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.176 seconds
[2024-11-11T00:53:34.469+0000] {processor.py:161} INFO - Started process (PID=59) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:53:34.473+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:53:34.489+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:53:34.488+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:53:37.665+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:53:37.664+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:53:38.984+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:53:39.075+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:53:39.074+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:53:39.082+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:53:39.082+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:53:39.093+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 4.655 seconds
[2024-11-11T00:54:09.394+0000] {processor.py:161} INFO - Started process (PID=92) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:54:09.395+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:54:09.398+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:54:09.398+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:54:10.799+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:54:10.798+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:54:11.030+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:54:11.044+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:54:11.043+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:54:11.049+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:54:11.049+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:54:11.055+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.673 seconds
[2024-11-11T00:54:41.393+0000] {processor.py:161} INFO - Started process (PID=124) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:54:41.398+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:54:41.400+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:54:41.399+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:54:42.375+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:54:42.374+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:54:42.610+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:54:42.622+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:54:42.622+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:54:42.627+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:54:42.627+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:54:42.633+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.247 seconds
[2024-11-11T00:55:13.014+0000] {processor.py:161} INFO - Started process (PID=142) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:55:13.021+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:55:13.034+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:55:13.033+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:55:14.914+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:55:14.914+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:55:15.562+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:55:15.602+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:55:15.602+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:55:15.610+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:55:15.610+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:55:15.622+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 2.645 seconds
[2024-11-11T00:55:45.897+0000] {processor.py:161} INFO - Started process (PID=167) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:55:45.900+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:55:45.907+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:55:45.906+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:55:46.907+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:55:46.906+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:55:47.181+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:55:47.196+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:55:47.196+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:55:47.202+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:55:47.202+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:55:47.210+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.318 seconds
[2024-11-11T00:56:17.561+0000] {processor.py:161} INFO - Started process (PID=192) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:56:17.566+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:56:17.572+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:56:17.572+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:56:18.906+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:56:18.906+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:56:19.170+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:56:19.192+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:56:19.191+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:56:19.204+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:56:19.204+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:56:19.212+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.668 seconds
[2024-11-11T00:59:44.654+0000] {processor.py:161} INFO - Started process (PID=59) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:59:44.660+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T00:59:44.668+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:59:44.667+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:59:47.682+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:59:47.678+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T00:59:48.875+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T00:59:48.973+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:59:48.972+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T00:59:48.981+0000] {logging_mixin.py:188} INFO - [2024-11-11T00:59:48.980+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T00:59:48.993+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 4.355 seconds
[2024-11-11T01:00:57.482+0000] {processor.py:161} INFO - Started process (PID=65) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:00:57.486+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:00:57.495+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:00:57.494+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:01:00.483+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:01:00.482+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:01:01.842+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:01:02.036+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:01:02.035+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:01:02.057+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:01:02.056+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:01:02.079+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 4.606 seconds
[2024-11-11T01:01:32.476+0000] {processor.py:161} INFO - Started process (PID=98) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:01:32.478+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:01:32.481+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:01:32.480+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:01:33.465+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:01:33.465+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:01:33.748+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:01:33.760+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:01:33.760+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:01:33.766+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:01:33.766+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:01:33.777+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.318 seconds
[2024-11-11T01:02:32.780+0000] {processor.py:161} INFO - Started process (PID=59) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:02:32.782+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:02:32.788+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:02:32.787+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:02:34.959+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:02:34.958+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:02:36.186+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:02:36.371+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:02:36.369+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:02:36.382+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:02:36.382+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:02:36.408+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 3.634 seconds
[2024-11-11T01:03:06.810+0000] {processor.py:161} INFO - Started process (PID=98) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:03:06.812+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:03:06.816+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:03:06.816+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:03:07.755+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:03:07.755+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:03:07.996+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:03:08.008+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:03:08.008+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:03:08.017+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:03:08.017+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:03:08.024+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.220 seconds
[2024-11-11T01:03:38.385+0000] {processor.py:161} INFO - Started process (PID=123) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:03:38.386+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:03:38.391+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:03:38.391+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:03:39.361+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:03:39.360+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:03:39.629+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:03:39.641+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:03:39.641+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:03:39.646+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:03:39.646+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:03:39.653+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.273 seconds
[2024-11-11T01:04:10.046+0000] {processor.py:161} INFO - Started process (PID=148) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:04:10.048+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:04:10.050+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:04:10.050+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:04:11.145+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:04:11.145+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:04:11.607+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:04:11.637+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:04:11.637+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:04:11.656+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:04:11.656+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:04:11.666+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.628 seconds
[2024-11-11T01:04:42.045+0000] {processor.py:161} INFO - Started process (PID=173) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:04:42.046+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:04:42.048+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:04:42.047+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:04:43.277+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:04:43.277+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:04:43.690+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:04:43.710+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:04:43.709+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:04:43.717+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:04:43.717+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:04:43.726+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.686 seconds
[2024-11-11T01:05:13.782+0000] {processor.py:161} INFO - Started process (PID=198) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:05:13.785+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:05:13.787+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:05:13.787+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:05:14.509+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:05:14.508+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:05:14.787+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:05:14.802+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:05:14.801+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:05:14.816+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:05:14.816+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:05:14.833+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.054 seconds
[2024-11-11T01:05:45.209+0000] {processor.py:161} INFO - Started process (PID=223) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:05:45.210+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:05:45.213+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:05:45.213+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:05:46.180+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:05:46.180+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:05:46.589+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:05:46.630+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:05:46.628+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:05:46.638+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:05:46.638+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:05:46.647+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.464 seconds
[2024-11-11T01:06:16.967+0000] {processor.py:161} INFO - Started process (PID=248) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:06:16.969+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:06:16.971+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:06:16.971+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:06:17.672+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:06:17.672+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:06:17.919+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:06:17.932+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:06:17.931+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:06:17.937+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:06:17.937+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:06:17.944+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.984 seconds
[2024-11-11T01:06:48.257+0000] {processor.py:161} INFO - Started process (PID=273) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:06:48.258+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:06:48.261+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:06:48.260+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:06:48.848+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:06:48.848+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:06:49.106+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:06:49.119+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:06:49.119+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:06:49.124+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:06:49.124+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:06:49.131+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.880 seconds
[2024-11-11T01:07:19.447+0000] {processor.py:161} INFO - Started process (PID=298) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:07:19.448+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:07:19.450+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:07:19.450+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:07:20.180+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:07:20.180+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:07:20.405+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:07:20.417+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:07:20.416+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:07:20.422+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:07:20.421+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:07:20.427+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.988 seconds
[2024-11-11T01:07:50.821+0000] {processor.py:161} INFO - Started process (PID=323) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:07:50.822+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:07:50.824+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:07:50.824+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:07:51.457+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:07:51.457+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:07:51.660+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:07:51.680+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:07:51.680+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:07:51.684+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:07:51.684+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:07:51.690+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.879 seconds
[2024-11-11T01:08:22.345+0000] {processor.py:161} INFO - Started process (PID=348) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:08:22.346+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:08:22.349+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:08:22.348+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:08:22.993+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:08:22.993+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:08:23.217+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:08:23.230+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:08:23.229+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:08:23.234+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:08:23.234+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:08:23.241+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.906 seconds
[2024-11-11T01:08:53.623+0000] {processor.py:161} INFO - Started process (PID=373) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:08:53.625+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:08:53.627+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:08:53.627+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:08:54.237+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:08:54.237+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:08:54.464+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:08:54.479+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:08:54.479+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:08:54.485+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:08:54.485+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:08:54.493+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.876 seconds
[2024-11-11T01:09:24.923+0000] {processor.py:161} INFO - Started process (PID=398) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:09:24.925+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:09:24.927+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:09:24.927+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:09:25.666+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:09:25.665+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:09:25.957+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:09:25.980+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:09:25.980+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:09:25.993+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:09:25.993+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:09:26.002+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.084 seconds
[2024-11-11T01:26:17.212+0000] {processor.py:161} INFO - Started process (PID=58) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:26:17.214+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:26:17.217+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:26:17.217+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:26:20.840+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:26:20.836+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:26:23.899+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:26:24.014+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:26:24.014+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:26:24.021+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:26:24.021+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:26:24.034+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 6.831 seconds
[2024-11-11T01:26:54.268+0000] {processor.py:161} INFO - Started process (PID=97) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:26:54.269+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:26:54.271+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:26:54.271+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:26:55.189+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:26:55.189+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:26:55.533+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:26:55.556+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:26:55.556+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:26:55.573+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:26:55.573+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:26:55.584+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.320 seconds
[2024-11-11T01:27:25.872+0000] {processor.py:161} INFO - Started process (PID=122) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:27:25.873+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:27:25.880+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:27:25.879+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:27:26.874+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:27:26.874+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:27:27.214+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:27:27.236+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:27:27.235+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:27:27.242+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:27:27.242+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:27:27.256+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.390 seconds
[2024-11-11T01:27:57.473+0000] {processor.py:161} INFO - Started process (PID=147) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:27:57.474+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:27:57.475+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:27:57.474+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:27:58.259+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:27:58.259+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:27:58.533+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:27:58.546+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:27:58.546+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:27:58.551+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:27:58.551+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:27:58.558+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.088 seconds
[2024-11-11T01:28:28.718+0000] {processor.py:161} INFO - Started process (PID=172) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:28:28.719+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:28:28.720+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:28:28.720+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:28:29.318+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:28:29.318+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:28:29.549+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:28:29.559+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:28:29.559+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:28:29.564+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:28:29.564+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:28:29.572+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.856 seconds
[2024-11-11T01:28:59.886+0000] {processor.py:161} INFO - Started process (PID=197) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:28:59.887+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:28:59.888+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:28:59.888+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:29:00.451+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:29:00.451+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:29:00.644+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:29:00.653+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:29:00.653+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:29:00.657+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:29:00.657+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:29:00.663+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.781 seconds
[2024-11-11T01:29:30.737+0000] {processor.py:161} INFO - Started process (PID=222) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:29:30.738+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:29:30.739+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:29:30.738+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:29:31.288+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:29:31.288+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:29:31.501+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:29:31.512+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:29:31.511+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:29:31.516+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:29:31.516+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:29:31.523+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.789 seconds
[2024-11-11T01:30:01.646+0000] {processor.py:161} INFO - Started process (PID=247) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:30:01.647+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:30:01.648+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:30:01.648+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:30:02.217+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:30:02.217+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:30:02.423+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:30:02.434+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:30:02.433+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:30:02.438+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:30:02.438+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:30:02.444+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.800 seconds
[2024-11-11T01:30:32.503+0000] {processor.py:161} INFO - Started process (PID=272) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:30:32.504+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:30:32.505+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:30:32.505+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:30:33.093+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:30:33.093+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:30:33.298+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:30:33.308+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:30:33.308+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:30:33.313+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:30:33.313+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:30:33.319+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.819 seconds
[2024-11-11T01:48:02.268+0000] {processor.py:161} INFO - Started process (PID=59) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:48:02.269+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:48:02.274+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:48:02.274+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:48:05.955+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:48:05.947+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:48:08.083+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:48:08.242+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:48:08.242+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:48:08.248+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:48:08.248+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:48:08.269+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 6.006 seconds
[2024-11-11T01:48:38.458+0000] {processor.py:161} INFO - Started process (PID=99) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:48:38.459+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:48:38.460+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:48:38.460+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:48:39.109+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:48:39.109+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:48:39.347+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:48:39.358+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:48:39.358+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:48:39.364+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:48:39.364+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:48:39.371+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.916 seconds
[2024-11-11T01:49:09.494+0000] {processor.py:161} INFO - Started process (PID=124) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:49:09.495+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:49:09.497+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:49:09.497+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:49:10.126+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:49:10.126+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:49:10.350+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:49:10.361+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:49:10.361+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:49:10.365+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:49:10.365+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:49:10.372+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.880 seconds
[2024-11-11T01:49:40.675+0000] {processor.py:161} INFO - Started process (PID=149) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:49:40.676+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:49:40.678+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:49:40.678+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:49:42.125+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:49:42.125+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:49:42.477+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:49:42.518+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:49:42.517+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:49:42.530+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:49:42.530+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:49:42.546+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.877 seconds
[2024-11-11T01:50:12.811+0000] {processor.py:161} INFO - Started process (PID=174) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:50:12.813+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:50:12.815+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:50:12.814+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:50:13.559+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:50:13.559+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:50:13.818+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:50:13.834+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:50:13.834+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:50:13.840+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:50:13.840+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:50:13.848+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.043 seconds
[2024-11-11T01:50:43.982+0000] {processor.py:161} INFO - Started process (PID=199) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:50:43.983+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T01:50:43.984+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:50:43.984+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:50:44.547+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:50:44.547+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T01:50:44.756+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T01:50:44.767+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:50:44.767+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T01:50:44.772+0000] {logging_mixin.py:188} INFO - [2024-11-11T01:50:44.772+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T01:50:44.778+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 0.800 seconds
[2024-11-11T04:14:58.622+0000] {processor.py:161} INFO - Started process (PID=101) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T04:14:58.634+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T04:14:58.639+0000] {logging_mixin.py:188} INFO - [2024-11-11T04:14:58.637+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T04:15:01.691+0000] {logging_mixin.py:188} INFO - [2024-11-11T04:15:01.691+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T04:15:02.993+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T04:15:03.385+0000] {logging_mixin.py:188} INFO - [2024-11-11T04:15:03.383+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T04:15:03.405+0000] {logging_mixin.py:188} INFO - [2024-11-11T04:15:03.405+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T04:15:03.423+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 4.842 seconds
[2024-11-11T04:15:34.239+0000] {processor.py:161} INFO - Started process (PID=164) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T04:15:34.255+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T04:15:34.265+0000] {logging_mixin.py:188} INFO - [2024-11-11T04:15:34.264+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T04:15:35.979+0000] {logging_mixin.py:188} INFO - [2024-11-11T04:15:35.978+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T04:15:36.306+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T04:15:36.317+0000] {logging_mixin.py:188} INFO - [2024-11-11T04:15:36.317+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T04:15:36.322+0000] {logging_mixin.py:188} INFO - [2024-11-11T04:15:36.322+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T04:15:36.328+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 2.111 seconds
[2024-11-11T04:16:06.554+0000] {processor.py:161} INFO - Started process (PID=226) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T04:16:06.555+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T04:16:06.558+0000] {logging_mixin.py:188} INFO - [2024-11-11T04:16:06.558+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T04:16:07.378+0000] {logging_mixin.py:188} INFO - [2024-11-11T04:16:07.378+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T04:16:07.638+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T04:16:07.649+0000] {logging_mixin.py:188} INFO - [2024-11-11T04:16:07.648+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T04:16:07.653+0000] {logging_mixin.py:188} INFO - [2024-11-11T04:16:07.653+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T04:16:07.659+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.122 seconds
[2024-11-11T04:16:37.766+0000] {processor.py:161} INFO - Started process (PID=289) to work on /opt/airflow/dags/dag_script_model.py
[2024-11-11T04:16:37.767+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script_model.py for tasks to queue
[2024-11-11T04:16:37.769+0000] {logging_mixin.py:188} INFO - [2024-11-11T04:16:37.769+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script_model.py
[2024-11-11T04:16:38.557+0000] {logging_mixin.py:188} INFO - [2024-11-11T04:16:38.557+0000] {plot.py:39} ERROR - Importing plotly failed. Interactive plots will not work.
[2024-11-11T04:16:38.777+0000] {processor.py:840} INFO - DAG(s) 'modeling_pipeline' retrieved from /opt/airflow/dags/dag_script_model.py
[2024-11-11T04:16:38.787+0000] {logging_mixin.py:188} INFO - [2024-11-11T04:16:38.786+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-11T04:16:38.790+0000] {logging_mixin.py:188} INFO - [2024-11-11T04:16:38.790+0000] {dag.py:3954} INFO - Setting next_dagrun for modeling_pipeline to None, run_after=None
[2024-11-11T04:16:38.796+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script_model.py took 1.044 seconds
